src/map/script.cpp

```diff
@@ -28105,6 +28152,10 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(setdialogpos, "ii"),
 	BUILDIN_DEF(setdialogpospercent, "ii"),
 
+	BUILDIN_DEF(checkperm, "i?"),
```

Comment:
Should we add aliases?
IE: "checkpermission", "addpermission", "removepermission"